src/hooks/use-page-errors.test.tsx

```diff
@@ -1,3 +1,86 @@
+import { renderHook, act } from "@testing-library/react-hooks";
+import { usePageErrors } from "./use-page-errors";
+import { ResultRecord } from "andculturecode-javascript-core";
+
 describe("usePageErrors", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React/issues/22", () => {});
+    describe("initialized", () => {
+        test("returns empty array", async () => {
+            // Arrange & Act
+            const { result } = renderHook(() => usePageErrors());
+
+            // Assert
+            expect(result.current.pageErrors).toBeEmpty();
+        });
+    });
+
+    describe("setPageErrors", () => {
+        test("when set with string array, pageErrors returns array", async () => {
+            // Arrange
+            const errorMessage = "Error Message";
+            const { result } = renderHook(() => usePageErrors());
+
+            // Act
+            act(() => {
+                result.current.setPageErrors([errorMessage]);
+            });
+
+            // Assert
+            expect(result.current.pageErrors.length).toBe(1);
+            expect(result.current.pageErrors[0]).toBe(errorMessage);
+        });
+    });
+
+    describe("handlePageLoadError", () => {
+        test("when error is string, pageErrors returns array", async () => {
```

Comment:
Just curious - do these test blocks need to be `async`? I'm not seeing any `await`s in here

Reply:
Absolutely not.  Good catch! Not sure how that got in there.